### PR TITLE
remove old api from python snippets and docs

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - remove compatibility Python API (__openvino.inference_engine__) and nGraph Python API from python snippets and some docs
 - Targeted PR: https://github.com/openvinotoolkit/openvino/pull/20908

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by sgolebiewski-intel — APPROVED** [review]

**Review by msmykx-intel — APPROVED** [review]

File: docs/snippets/ov_preprocessing_migration.py [hunk 1]
```diff
@@ -1,102 +0,0 @@
-# Copyright (C) 2018-2023 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-#
-
-from utils import get_model, get_ngraph_model
-
-#! [ov_imports]
-from openvino import Core, Layout, Type
-from openvino.preprocess import ColorFormat, PrePostProcessor, ResizeAlgorithm
-#! [ov_imports]
```
Comment (jiwaszki) [inline docs/snippets/ov_preprocessing_migration.py:10]: Here you removed also API 2.0 imports and code. Please revert it.

Comment (akuporos) [reply docs/snippets/ov_preprocessing_migration.py:10]: this is migration guide.
We have two another snippets which show how to use preprocessing and properties.

File: docs/snippets/ov_properties_migration.py [hunk 2]
```diff
@@ -1,91 +0,0 @@
...
-        return 0
-
-    # ! [core_compile_model]
-    compiled_model = core.compile_model(model=model, device_name="MULTI", config=
-        {
-            device.priorities: "GPU,CPU",
-            hints.performance_mode: hints.PerformanceMode.THROUGHPUT,
-            hints.inference_precision: ov.Type.f32
-        })
-    # ! [core_compile_model]
```
[the remark this answers is not in the document]
Comment (akuporos) [reply docs/snippets/ov_properties_migration.py:32]: this is the migration guide.
We have two another snippets which show how to use preprocessing and properties.

File: docs/articles_en/documentation/openvino_legacy_features/api_2_0_transition_guide/preprocessing.md [hunk 3]
```diff
@@ -37,24 +37,6 @@ The snippets assume that you need to preprocess a model input with the ``tensor_
...
-    :fragment: imports
-
-
-**API 2.0**
-
-
-.. doxygensnippet:: docs/snippets/ov_preprocessing_migration.py
-    :language: python
-    :fragment: ov_imports
-
```
[the remark this answers is not in the document]
Comment (akuporos) [reply docs/articles_en/documentation/openvino_legacy_features/api_2_0_transition_guide/preprocessing.md:56]: No, as this is transition guide from old API to the new one

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**